src/bin/wc.rs

```diff
@@ -103,6 +103,10 @@ struct Flags {
     count_bytes: bool,
 }
 
+fn u64_num_digits(val: u64) -> usize {
+    (val as f32).log10() as usize + 1
```

Comment:
What's wrong with this implementation? It works: https://stackoverflow.com/questions/3068397/finding-the-length-of-an-integer-in-c